### PR TITLE
Update Compat section as the Reporting API is now in bcd

### PR DESCRIPTION
**Pull request by teoli2003**
Follow-up of #16066

As the Reporting API has been added to BCD in mdn/browser-compat-data#10925, we can display the table (from tomorrow I think).

(Keeping this in draft until the new BCD package is actually published)

### CODE REVIEW

File: files/en-us/web/api/interventionreportbody/id/index.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ const observer = new ReportingObserver((reports, observer) => {
 
 ## Browser compatibility
 
-This feature is not yet available by default in any released browser. It can be activated in Firefox by setting `dom_reporting_enabled` to `true` and in Chrome if you [enable this experimental feature](https://web.dev/reporting-api/#use-devtools).
+{{Compat}}.
```
Comment (Josh-Cena) [inline files/en-us/web/api/interventionreportbody/id/index.md:45]:
```suggestion
{{Compat}}
```

**Review by Josh-Cena — COMMENTED** [review]
Is this ready?

**Comment by teoli2003** [comment]
I completely forgot this; yes, it should.

File: files/en-us/web/api/crashreportbody/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ tags:
   - Interface
   - Reference
   - Reporting API
-spec-urls: https://wicg.github.io/crash-reporting/#crashreportbody
+browser-compat: api.CrashReportBody
```
Comment (Josh-Cena) [inline files/en-us/web/api/crashreportbody/index.md:12]:
```suggestion
browser-compat: api.CrashReportBody
```

**Review by Josh-Cena — APPROVED** [review]
LGTM

**Pull request merged**